cmd/cql/internal/help.go

```diff
@@ -0,0 +1,37 @@
+package internal
+
+import (
+	"fmt"
+	"runtime"
+)
+
+const name = "cql"
+
+var (
+	// Version of command, set by main func of version
+	Version = "unknown"
+)
+
+var CmdVersion = &Command{
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var CmdVersion should have comment or be unexported